Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/StubIr.kt

```diff
@@ -341,6 +344,11 @@ sealed class PropertyAccessor : FunctionalStub {
                 val constant: ConstantStub? = null
         ) : Getter()
 
+        class ConstructorParameter(
+                val constructorParameter: FunctionParameterStub,
+                override val annotations: List<AnnotationStub> = emptyList()
+        ) : Getter()
```

Comment:
This code can be read as "constructor parameter is a getter". Is this intended? What does this mean?

Reply:
Ah, I see. Maybe `GetConstructorParameter`?
Also, it is initializer, and you can't actually use a constructor parameter in a getter.

Reply:
Agreed. What do you think about adding `backingField: ValueStub?` field to `PropertyStub` and adding `PropertyAccessor.Getter.ReadBackingField` instead of `ConstructorParameter `

Reply:
I guess `GetConstructorParameter` is ok for now, so maybe later.